game/resources/sfx.lua

```diff
@@ -0,0 +1,49 @@
+local Sfx = {}
+
+local Source = Class{}
```

Comment:
Please use parenthesis and avoid lua's sugar function calls

Reply:
but we use it all the time! Look at our requires for instance

Reply:
Ok but we _only_ use it for requires

Reply:
Okay